### ISSUE TITLE
I have downloaded the training model, why is it wrong?

### ISSUE THREAD

**Issue by wuheng199112068** [fangchangma/sparse-to-dense#21]
(base) by1706143@BY1706143:/tmp/mozilla_by17061430/sparse-to-dense-master$ th main.lua -testOnly true -dataset nyudepthv2 -inputType rgbd -nSample 0  -rep linear -encoderType conv -decoderType deconv2 -criterion l1 -LR 0.01  -pretrain true
	
error: trained model does not exist: /tmp/mozilla_by17061430/sparse-to-dense-master/results/nyudepthv2.input=rgbd.nsample=0.rep=linear.encoder=conv.decoder=deconv2.criterion=l1.lr=0.01.bs=16.pretrained=true/model_best.t7	
Usage: [options]

Sparse-to-Dense Training script
See https://github.com/fangchangma/sparse-to-dense for details

Train Options:
  -dataset            Options: nyudepthv2 | kitti [nyudepthv2]
  -inputType          Options: rgb | rgbd | d | g | gd [rgb]
  -nSample            average number of depth samples [0]
  -criterion          Options: l1 | l2 | berhu [l1]
  -pretrain           use pretrained model [false]
  -rep                Representation of depth. Options: linear | log | inverse [linear]
  -encoderType        Options: conv | channeldrop | depthsep [conv]
  -decoderType        Options: deconv2 | deconv3 | upconv | upproj [upproj]
  -manualSeed         Manually set RNG seed [0]
  -nGPU               Number of GPUs to use by default [1]
  -backend            Options: cudnn | cunn [cudnn]
  -cudnn              Options: fastest | default | deterministic [fastest]
  -gen                Path to save generated files [gen]
  -precision          Options: single | double | half [single]
  -nThreads           number of data loading threads [2]
  -nEpochs            Number of total epochs to run [20]
  -epochNumber        Manual epoch number (useful on restarts) [1]
  -batchSize          mini-batch size (1 = pure stochastic) [16]
  -testOnly           Run on validation set only [false]
  -tenCrop            Ten-crop testing [false]
  -resume             Path to directory containing checkpoint [none]
  -LR                 initial learning rate [0.01]
  -momentum           momentum [0.9]
  -weightDecay        weight decay [0.0001]
  -recomputeBatchNorm recompute batch norm statistics [false]
  -shortcutType       Options: A | B | C []
  -optimState         Path to an optimState to reload from [none]
  -optnet             Use optnet to reduce memory usage [true]
  -nClasses           Number of classes in the dataset [0]

Thank you for such great code.I have downloaded the training model, why is it wrong?
